tests/bluetooth/bsim_bt/bsim_test_audio/src/has_test.c

```diff
@@ -18,6 +18,15 @@ const char *test_preset_name_1 = "test_preset_name_1";
 const char *test_preset_name_5 = "test_preset_name_5";
 const enum bt_has_properties test_preset_properties = BT_HAS_PROP_AVAILABLE;
 
+static int preset_select(uint8_t index, bool sync)
+{
+	return 0;
+}
+
+static const struct bt_has_preset_ops preset_ops = {
+	.select = preset_select,
+};
```

Comment:
Is the client selecting a preset? Should we verify that?

Reply:
I will extend the tests in suceeding PR